src/lib/logic/utils/errors.ts

```diff
@@ -18,6 +30,8 @@ export class FormError extends Error {
 
     this.name = "FormError";
     this.title = title;
+    this.message = message,
+    this.reason = reason;
```

Comment:
🚫 **[eslint]** <[@typescript-eslint/no-unused-expressions](https://typescript-eslint.io/rules/no-unused-expressions)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Expected an assignment or function call and instead saw an expression.